src/app/githook/index.js

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs'
+import path from 'path'
+import { actionText, dim } from 'src/utils/log-style'
+
+function copyFiles() {
+  const templates = path.join(__dirname, '../src/app/githook/templates')
+  const hookPath = path.join(templates, 'pre-push')
+
+  fs.copyFileSync(hookPath, '.git/hooks/pre-push')
+}
+
+export async function githook() {
+  copyFiles()
+}
```

Comment:
This is what I'm talking about, if I run `code-config githook` the user doesn't get any feedback of what's happening and it only works with ESLint.

Reply:
yep, I can add something to include style lint

Reply:
yup, good point